app/assets/stylesheets/responsive.scss

```diff
@@ -66,6 +71,10 @@ $navCollapseColor: #777;
 
     tr { border: 1px solid #ccc; }
 
+    .first-in-bundle {
```

Comment:
Do we need this? I think the definition inside of tables.scss should cover this.